www/index.html

```diff
@@ -6,7 +6,8 @@
 		<meta name="description" content="HackCWRU 2017 hosted at Case Western Reserve University. Sign up for email updates!">
 		<meta name="author" content="HackCWRU">
 		<link href="site.css" rel="stylesheet">
-		<link href='https://fonts.googleapis.com/css?family=Source+Sans+Pro:200' rel='stylesheet' type='text/css'>
+		
+		<link href='http://fonts.googleapis.com/css?family=Open+Sans' rel='stylesheet' type='text/css'>
```

Comment:
I dont actually know what that was, I didnt notice that when I was doing the editting, so ill just  add the ":200" after "Open+Sans"